### DISCUSSION TITLE
Can the server be configured to support mod？

### DISCUSSION THREAD

**Discussion by Fantast616** [Impostor/Impostor#462] [Q&A] [answered]
I and my friends want to play the mod 3.4.2 from this website:  https://www.amonguscn.cn/ , we use the 3.4.2 version of the mod.
I have already deployed with docker on my private server( as the docs described), and a problem happened when we want to play the game in my private server( we have already set up the server config in our client). i can successfully create a room, and can successfully get the room code, but when my friends enter the code, the client will show "you cheated xxxxxxxxx",and then not only he can't enter my room ,but also i fell out of this room. 

then we tried again, we disable the mod,and use the origin client version, then we can play successfully. so i think it is because the mismatch between the client and server, so i want to ask can i configure the server files to support my client mod? if can , how to do that ? thank you !

**Comment by miniduikboot** [comment, marked as answer]
Hi there, if you play with mods you need to disable the anticheat in config.json. In your default config file, you'd change [this line](https://github.com/Impostor/Impostor/blob/master/src/Impostor.Server/config.json#L9) to `false`, but you can also set the environment variable `IMPOSTOR_AntiCheat__Enabled=false`

**Reply by Fantast616** [reply]
yep, i set the anticheat = false, then i success! thank you very much!
